bufferserver/src/main/java/com/datatorrent/bufferserver/internal/DataList.java

```diff
@@ -845,6 +854,30 @@ protected void release(boolean wait)
       }
     }
 
+    private boolean canRelease(boolean writer)
```

Comment:
IMO, renaming release() is less desirable. Note that acquire() does not necessarily brings block from the disk. It may simply increment ref count and return. The release() is a counterpart of the acquire() method and decrements ref count. The naming convention for the release() is similar to IUnknown COM interface, where release() also decrements ref count and if it drops to 0, invokes destructor. Let's rename to canEvict().